avm/res/web/hosting-environment/import.bicep

```diff
@@ -0,0 +1,324 @@
+metadata name = 'App Service Environments'
```

Comment:
Why is there an `import.bicep`? :D Is that a leftover? I did review it regardless, but ... it shouldn't be here, I think

Reply:
Ha! Yes, that was just used for troubleshooting from CARML repo. Removed. :)